### PR TITLE
Remove return from transferFrom

### PR DESCRIPTION
**Pull request by payvint**
Remove return cover into try/catch
Add test

### CODE REVIEW

**Review by sergiy-skalelabs — APPROVED** [review]

**Review by sergiy-skalelabs — APPROVED** [review]

**Review by cstrangedk — COMMENTED** [review]
lgtm

**Review by cstrangedk — APPROVED** [review]

**Review by yavrsky — APPROVED** [review]

File: proxy/contracts/test/ERC20IncorrectTransfer.sol [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    function name() external view returns (string memory);
+    function decimals() external view returns (uint8);
+    function symbol() external view returns (string memory);
+    function totalSupply() external view returns (uint256);
+    function balanceOf(address account) external view returns (uint256);
+    function allowance(address owner, address spender) external view returns (uint256);
+}
+
+
+contract ERC20IncorrectTransfer is IERC20IncorrectTransfer {
```
Comment (DimaStebaev) [inline proxy/contracts/test/ERC20IncorrectTransfer.sol:22]: If it's not possible to extend ERC20 I recommend to aggregate it.

File: proxy/contracts/test/ERC20TransferWithoutReturn.sol [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    function decreaseAllowance(address spender, uint256 subtractedValue) external returns (bool);
+    function name() external view returns (string memory);
+    function decimals() external view returns (uint8);
+    function symbol() external view returns (string memory);
+    function totalSupply() external view returns (uint256);
+    function balanceOf(address account) external view returns (uint256);
+    function allowance(address owner, address spender) external view returns (uint256);
+}
+
+contract ERC20TransferWithoutReturn is IERC20TransferWithoutReturn {
```
Comment (DimaStebaev) [inline proxy/contracts/test/ERC20TransferWithoutReturn.sol:21]: If it's not possible to extend ERC20 I recommend to aggregate it.

**Comment by payvint** [comment]
Unsecure fix

**Comment by payvint** [comment]
Unsecure fix

**Pull request closed without merging**